lib/msf/base/serializer/readable_text.rb

```diff
@@ -5,13 +5,65 @@ module Serializer
 # This class formats information in a plain-text format that
 # is meant to be displayed on a console or some other non-GUI
 # medium.
-class ReadableText
+class ReadableText < Rex::Ui::Text::Output::Stdio
```

Comment:
Rex::Ui::Text::Output::Stdio contains methods necessary for verifying that a given terminal environment supports formatting.